### DISCUSSION TITLE
Canva Pró

### DISCUSSION THREAD

**Discussion by MatheusArv** [community/community#54563] [GitHub Education]
Criei minha conta institucional e loguei no GitHub com ela mas não consigo achar o app do Canva no GitHub para liberar o Canva Pró. Alguém sabe como achar o aplicativo do Canva aqui? Já tentei nas barras de pesquisa e não achei.

**Comment by GeoffreyFClark** [comment]
O aplicativo pode ser baixado do site deles, não do GitHub. Se você quiser, o GitHub tem uma organização do Canva que você pode conferir: https://github.com/Canva

Você pode obter o Canva Pro gratuitamente se for estudante ou educador e se inscrever no site https://www.canva.com/education/.

**Comment by MarcioBFS** [comment]
tambem nao consegui

**Comment by samus-aran** [comment]
Hi @MatheusArv  👋 ! Thanks for the great question about Global Campus Benefits.

We no longer provide the Canva benefit as part of the Student Developer Pack. You can check out all the Student Developer Pack's latest offers and partners available [here](https://education.github.com/pack?sort=az#offers).
